modules/openlmis-web/src/main/webapp/public/js/distribution/model/epi-inventory.js

```diff
@@ -10,18 +10,47 @@
 
 function EpiInventory(epiInventory) {
   $.extend(true, this, epiInventory);
+
   $(this.lineItems).each(function (index, lineItem) {
     lineItem.existingQuantity = this.existingQuantity || {};
     lineItem.spoiledQuantity = this.spoiledQuantity || {};
   });
 
   var mandatoryFields = ['existingQuantity', 'deliveredQuantity', 'spoiledQuantity'];
+  var nrFields = ['existingQuantity', 'spoiledQuantity'];
+
+  function init() {
+    var fieldsStatusCount = countNRStatus(this.lineItems);
+    this.notRecordedApplied = (fieldsStatusCount.notRecorded > fieldsStatusCount.recorded);
+  }
+
+  function countNRStatus(lineItems) {
```

Comment:
I like the fact that you moved this logic into a function. I notice that a nearly identical function is also defined in epi-use.js, though. Would it be possible to define countNRStatus(lineItems) just once, in a shared location, and reuse the function?